toolkit/components/uniffi-bindgen-gecko-js/src/templates/js/wrapper.jsm

```diff
@@ -0,0 +1,11 @@
+// This file was autogenerated by some hot garbage in the `uniffi` crate.
+// Trust me, you don't want to mess with it!
+
+"use strict";
+
+{% include "Helpers.jsm" %}
+
```

Comment:
we can probably parallelize here by having seperate files here (Enums.jsm, Errors.jsm, etc) if we need to - I'll get to them eventually if we don't need to, they're not too bad